VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitFragment.java

```diff
@@ -14,6 +14,8 @@
 public abstract class CourseUnitFragment extends BaseFragment implements PageViewStateCallback, RunnableCourseComponent {
     public interface HasComponent {
         CourseComponent getComponent();
+        boolean navigateNextComponent();
```

Comment:
At the moment, nothing. When I first made it, it mattered. I shifted the
logic around and now nothing needs it, but I kept in the return variable.
Do you want to make it void? That’s a simple change.

On Wed, Aug 3, 2016 at 1:46 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitFragment.java
> <https://github.com/edx/edx-app-android/pull/761#discussion_r73379174>:
>
> > @@ -14,6 +14,8 @@
> >  public abstract class CourseUnitFragment extends BaseFragment implements PageViewStateCallback, RunnableCourseComponent {
> >      public interface HasComponent {
> >          CourseComponent getComponent();
> > +        boolean navigateNextComponent();
>
> Let me clarify, what calls this that requires a boolean value.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/761/files/64c1f08c0f3651fb25188c643146d7accc62968d#r73379174>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR10T58gdPxJXsJZ53wuFxqIU7kR8yCcks5qcNPvgaJpZM4JCZsX>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX